website/static/js/fangorn.js

```diff
@@ -184,6 +186,59 @@ var uploadRowTemplate = function(item){
 };
 
 /**
+ * Returns custom icons for OSF depending on the type of item. Used for non-file icons.
+ * @param {Object} item A Treebeard _item object. Node information is inside item.data
+ * @this Treebeard.controller
+ * @returns {Object}  Returns a mithril template with the m() function.
+ */
+function resolveIconHelper(item) {
```

Comment:
No need to add "Helper" to function names. Use verbs when possible, e.g. `resolveIcon`. 

Will make this change when I merge.